Snowflake.Data/Core/RestResponse.cs

```diff
@@ -27,60 +37,51 @@ class NullDataResponse : BaseRestResponse
         internal object data { get; set; }
     }
 
-    internal class AuthnResponse : BaseRestResponse
+    internal class AuthenticatorResponse : BaseRestResponse
     {
         [JsonProperty(PropertyName = "data")]
-        internal AuthnResponseData data { get; set; }
+        internal AuthenticatorResponseData data { get; set; }
     }
-    
+    internal class LoginResponse : BaseRestResponse
+    {
+        [JsonProperty(PropertyName = "data")]
+        internal LoginResponseData data { get; set; }
+    }
+
     internal class RenewSessionResponse : BaseRestResponse {
 		[JsonProperty(PropertyName = "data")]
 		internal RenewSessionResponseData data { get; set; }
 	}
 
-    internal class AuthnResponseData
+    internal class LoginResponseData
     {
         [JsonProperty(PropertyName = "token", NullValueHandling = NullValueHandling.Ignore)]
         internal string token { get; set; }
 
-        [JsonProperty(PropertyName = "validityInSeconds", NullValueHandling = NullValueHandling.Ignore)]
-        internal Int16 validityInSeconds { get; set; }
-
         [JsonProperty(PropertyName = "masterToken", NullValueHandling = NullValueHandling.Ignore)]
         internal string masterToken { get; set; }
 
-        [JsonProperty(PropertyName = "masterValidityInSeconds", NullValueHandling = NullValueHandling.Ignore)]
-        internal Int16 masterTokenValidityInSeconds { get; set; }
-
-        [JsonProperty(PropertyName = "displayUserName", NullValueHandling = NullValueHandling.Ignore)]
-        internal string displayUserName { get; set; }
-
         [JsonProperty(PropertyName = "serverVersion", NullValueHandling = NullValueHandling.Ignore)]
         internal string serverVersion { get; set; }
 
-        [JsonProperty(PropertyName = "firstLogin", NullValueHandling = NullValueHandling.Ignore)]
-        internal bool firstLogin { get; set; }
-
-        [JsonProperty(PropertyName = "healthCheckInterval", NullValueHandling = NullValueHandling.Ignore)]
-        internal Int16 healthCheckInterval { get; set; }
-
-        [JsonProperty(PropertyName = "newClientForUpgrade", NullValueHandling = NullValueHandling.Ignore)]
-        internal string newClientForUpgrade { get; set; }
-
-        [JsonProperty(PropertyName = "sessionId", NullValueHandling = NullValueHandling.Ignore)]

```

Comment:
Basically I agree to remove unused parameters for now. Later when we need it, we can put them back.